app/views/users/index.html.erb

```diff
@@ -1,3 +1,18 @@
-<div>
-     <h1> Here is a list of all users <h1>
-<div />
\ No newline at end of file
+<section class="show-user">
+   <div class="card" style="width: 18rem;">
+      <ul class="users_list">
+          <% @users.each do |user| %>
+               <li class = 'user' >
+               <% if user.photo %>
+                    <img class="card-img-top" src='<%= user.photo %>' alt='<%= user.name %> thumbnail' />
+               <% end %>
+                    <div class='card-body'>
+                         <h5> Username: <%= user.name%></h5>
+                         <p class="card-text">Number of Posts: <%= user.posts_counter %></p>
+                          <%= link_to 'View user posts', user_path(user), class: "btn btn-primary text-center"  %>
+                    </div>
+               </li>
+          <% end %>
+     </ul>
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure the app/views/users/index.html.erb view pag match the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png) as require for the project

Your design
![image](https://user-images.githubusercontent.com/81932478/214280079-98f53f7b-ead9-4747-b623-236574ec1434.png)

Given Wireframe
![image](https://user-images.githubusercontent.com/81932478/214280196-95888689-9fc6-4ff1-a88c-f1e3aca6f652.png)